pf4j/src/main/java/org/pf4j/Extension.java

```diff
@@ -34,4 +34,13 @@
 
     int ordinal() default 0;
 
+    /**
+     * An array of extension points, that are implemented by this extension.
+     * This explicit configuration overrides the automatic detection of extension points in the
+     * {@link org.pf4j.processor.ExtensionAnnotationProcessor}.
```

Comment:
I think that here we must add some explanation lines when to use this attribute (your approach with Adapter is very good as example - also my example from [Pippo](https://github.com/pippo-java/pippo/blob/b446c6debc8d9136c2b3e9ff4a8ee247dcdaf50f/pippo-server-parent/pippo-undertow/src/main/java/ro/pippo/undertow/UndertowServer.java#L63) is very clear as use case).

Reply:
I've made some changes to the documentation and [updated my pull request](https://github.com/pf4j/pf4j/pull/265/commits/0de925f7532c470e39c5460626f88de4b9cf3a8d). From my point of view it would be better to document this attribute in the [official documentation](https://pf4j.org/). The Javadoc could contain a link, that points to the corresponding page of the documentation website.

It's much easier to document those kind of issues together with images in the official documentation compared to a somewhat limited javadoc. Also I guess most users would expect to find those kind of information in the official documentation instead of the Javadocs.

Because the new attribute is currently not documented yet, I've just added a link to issue #264 to describe the use case. Maybe you can replace that link later or you can add another link to Pippo.